aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/client/RdsClientProvider.java

```diff
@@ -54,4 +55,9 @@ public RdsClient getClient() {
     public RdsClient getClientForApiVersion(@NonNull final String apiVersion) {
         return setUserAgentAndApiVersion(setHttpClient(RdsClient.builder()), apiVersion).build();
     }
+
+    public RdsClient getClientForRegion(@NonNull final String region) {
+        final Region sdkRegion = Region.of(region);
+        return setUserAgent(setHttpClient(RdsClient.builder().region(sdkRegion))).build();
+    }
```

Comment:
That would not work for the versioned API. Would that concern us?

Reply:
We exclusively use v12 requests for Create/Modify calls if DBSecurityGroup is present. It should not affect other calls.